packages/dot-agent-kit/src/dot_agent_kit/context.py

```diff
@@ -1,125 +1,170 @@
 """Application context with dependency injection.
 
-This module provides the canonical context creation pattern for dot-agent-kit,
-following the architecture established in erk's core/context.py.
+This module provides the canonical context for dot-agent-kit.
 
-The DotAgentContext dataclass holds all dependencies (GitHub integrations, config)
-and is created once at CLI entry point, then threaded through the application.
+The DotAgentContext is now an alias for erk_shared.context.ErkContext, providing
+a unified context for both erk and dot-agent-kit operations.
+
+Migration Notes:
+- DotAgentContext.github_issues -> ctx.issues (renamed for consistency)
+- DotAgentContext.repo_root -> ctx.repo_root (preserved as property)
+- All other fields remain the same
 """
```

Comment:
this file should be gone. change all callsites to call something else (move any code in here as necessary)